colorcoder/utils/EnumHelper.java

```diff
@@ -0,0 +1,22 @@
+package colorcoder.utils;
+
+import java.util.Arrays;
+
+import colorcoder.Color;
+
+/**
+ * @author {@literal Jayaram Naveenkumar (jayaram.naveenkumar@in.bosch.com)}
+ */
+public final class EnumHelper {
+
+    private EnumHelper() { }
+
+    // Generic return syntax is copy pasted from internet
```

Comment:
Does this comment add any value to the code readability
what does E and P represents ?

is EnumHelper is the valid name ?

Reply:
yup comment doesn't make sense, Modified